.github/workflows/examples-nrfconnect.yaml

```diff
@@ -131,7 +131,7 @@ jobs:
                     /tmp/bloat_reports/
             - name: Build example nRF Connect SDK All Clusters App on nRF52840 DK
               run: |
-                  scripts/examples/nrfconnect_example.sh all-clusters-app nrf52840dk_nrf52840 -DCONF_FILE=prj_dfu.conf
+                  scripts/examples/nrfconnect_example.sh all-clusters-app nrf52840dk_nrf52840
```

Comment:
This does not seem related to this PR.  And this change has already happened on master...